crates/ruff_annotate_snippets/src/renderer/display_list.rs

```diff
@@ -1089,13 +1111,14 @@ fn format_label(
     result
 }
 
-fn format_snippet(
-    snippet: snippet::Snippet<'_>,
+fn format_snippet<'m>(
+    snippet: snippet::Snippet<'m>,
     is_first: bool,
     has_footer: bool,
     term_width: usize,
     anonymized_line_numbers: bool,
-) -> DisplaySet<'_> {
+    cut_indicator: &'static str,
```

Comment:
Huh, surprising that `&'static` impacts the `'_` lifetime.

Reply:
Yeah I was a little surprised too.